### PR TITLE
appveyor.yml: Use curl again

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: .misc/appveyor.yml [hunk 1]
```diff
@@ -24,6 +24,10 @@ branches:
     - /^sils\/.*/
 
 install:
+  # curl exists in this path. See this discussion for more details:
+  # http://help.appveyor.com/discussions/problems/6312-curl-command-not-found
+  - "SET PATH=C:\\Program Files\\Git\\mingw64\\bin;%PATH%"
```
Comment (jayvdb) [inline .misc/appveyor.yml]: The new addition should be after the existing PATH, not before.
And it appears we don't need those quotes around the command.
If we remove them, we dont need the backslash escapes which appear in GitHub as syntax errors.
c.f. https://github.com/jayvdb/coala/commit/c5aadba1f76b5963e5b759f1ba403eb2cfbfd45b and https://ci.appveyor.com/project/jayvdb17618/coala/build/1.0.15

**Comment by adtac** [comment]
I just cherry-picked your commit ¯\\\_(ツ)_/¯

**Comment by adtac** [comment]
@jack17529 there aren't any associated issues.

**Comment by adtac** [comment]
@jayvdb any reason this is held up? (aside from rebases)

I basically cherry-picked your commit.

**Comment by Nosferatul** [comment]
ack 3e3cbdb

**Comment by Nosferatul** [comment]
@adtac could you rebase?

**Comment by adtac** [comment]
@Nosferatul gladly.

Done :+1:

**Comment by adtac** [comment]
reack https://github.com/coala/coala/pull/3965/commits/492407f9d15a2ac7db5bc24265f8559a959e49b6

**Comment by Nosferatul** [comment]
@rultor merge

**Comment by Nosferatul** [comment]
@rultor merge

**Comment by adtac** [comment]
@Nosferatul patience :) rultor takes its own sweet time to respond :3

**Comment by Nosferatul** [comment]
@adtac yea, right :)

**Pull request merged**